synapse/models/orgs.py

```diff
@@ -64,8 +64,39 @@ def getModelDefs(self):
                 ('ou:conference:event:attendee', ('comp', {'fields': (('conference', 'ou:conference:event'), ('person', 'ps:person'))}), {
                     'doc': 'Represents a person attending a conference event represented by an ou:conference:event node.',
                 }),
+                ('ou:goal', ('guid', {}), {
+                    'doc': 'An assessed or stated goal of an org',
+                }),
+                ('ou:campaign', ('guid', {}), {
+                    'doc': 'Represents an orgs activity in persuit of a goal',
+                }),
             ),
             'forms': (
+                ('ou:goal', {}, (
+                    ('name', ('str', {}), {
+                        'doc': 'A terse name for the goal',
+                    }),
+                    ('type', ('str', {}), {
```

Comment:
strip / lower ?

Reply:
Decided against in case somebody wants to use them in titles/headers.